main.scss

```diff
@@ -23,7 +23,10 @@
 @import 'scss/wrapped_general';
 
 @if $o-typography-is-silent == false {
-	@include oFontsIncludeAll();
+
+	@if $o-typography-load-fonts == true {
```

Comment:
Yup, just making it more versatile

On Mon, 23 May 2016 at 11:45 Rhys Evans <notifications@github.com> wrote:

> In main.scss
> <https://github.com/Financial-Times/o-typography/pull/72#discussion_r64202205>
> :
>
> > @@ -23,7 +23,10 @@
> >  @import 'scss/wrapped_general';
> >
> >  @if $o-typography-is-silent == false {
> > -	@include oFontsIncludeAll();
> > +
> > +	@if $o-typography-load-fonts == true {
>
> That would mean there's a possibility, in silent mode, to output the
> @font-face css
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/o-typography/pull/72/files/c4d7fec83e709c216599cec56a4ff5beb9397ac7#r64202205>
>

-- 

------------------------------

*This email was sent by a company owned by Financial Times Group Limited 
("FT Group <http://aboutus.ft.com/corporate-information/#axzz3rajCSIAt>"), 
registered office at Number One Southwark Bridge, London SE1 9HL.  
Registered in England and Wales with company number 879531. This e-mail may 
contain confidential information. If you are not the intended recipient, 
please notify the sender immediately, delete all copies and do not 
distribute it further.  It could also contain personal views which are not 
necessarily those of the FT Group.  We may monitor outgoing or 
incoming emails as permitted by law.*